recommended_scripts/vsx_health_monitor/README.md

```diff
@@ -0,0 +1,20 @@
+## Summary
+
+VSX Health monitoring
+
+## Supported Software Versions
+
+Script Version 4.0: ArubaOS-CX 10.13.1000 Minimum
+
+## Supported Platforms
+
+Script Version 4.0: 64xx, 8320, 8325, 8400
```

Comment:
No support of 8100 / 8360 ?

Reply:
As far as I know, no.  I will check with NTL.

Reply:
I checked with NTL and these platforms will be added, so I will add them to the list.